src/brain/README.md

```diff
@@ -1,3 +1,18 @@
 # brain
 
-The brain contains modules that listen to webhook events from GitHub and Slack. These modules are autoloaded by `@utils/loadBrain`.
+The brain contains modules that listen to webhook events from GitHub, Slack, and GoCD. These modules are autoloaded by `@utils/loadBrain`.
```

Comment:
Should we include options automator in this list too?

Reply:
Yes good idea

Reply:
actually no options automator is a webhook but its not in the brain module.

Reply:
actually no its not part of brain